src/test/pythonEnvironments/discovery/locators/pipEnvHelper.unit.test.ts

```diff
@@ -0,0 +1,273 @@
+import * as assert from 'assert';
+import * as path from 'path';
+import * as sinon from 'sinon';
+import * as platformApis from '../../../../client/common/utils/platform';
+import * as externalDependencies from '../../../../client/pythonEnvironments/common/externalDependencies';
+import { _getAssociatedPipfile, isPipenvEnvironment, isPipenvEnvironmentRelatedToFolder } from '../../../../client/pythonEnvironments/discovery/locators/services/pipEnvHelper';
+
+suite('Pipenv helper', () => {
+    suite('Method isPipenvEnvironmentRelatedToFolder()', async () => {

```

Comment:
FWIW, it's a function rather than a method.  If it were me I'd just drop that altogether:

```suggestion
    suite('isPipenvEnvironmentRelatedToFolder()', async () => {
```